net.devolutions.RDM.yml

```
@@ -0,0 +1,38 @@
+app-id: net.devolutions.RDM
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: remotedesktopmanager
+tags: [proprietary]
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --filesystem=host
+   
+modules:
+  - name: rdm
+    buildsystem: simple
+    build-commands:
+      - ar x RemoteDesktopManager.deb
+      - tar xf data.tar.xz
+      - mv usr/ /app/
```

You should do this instead:

```suggestion
      - mv usr/* /app/
```

(other changes needed to match this)